client-common/src/main/java/org/apache/cassandra/sidecar/common/response/GossipStatusResponse.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.common.response;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Response for GET gossip status API
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class GossipStatusResponse
+{
+    private final String gossipRunningStatus;
+
+    /**
+     * Constructs a response with gossip status
+     * @param gossipRunningStatus running status of gossip
+     */
+    public GossipStatusResponse(@JsonProperty("gossipRunningStatus") String gossipRunningStatus)
```

Comment:
I think `status` should be sufficient. We don't have a precedent for qualifying the field name like this.

Reply:
Just calling it 'status' can be mistaken with STATUS field of gossip info, as we discussed before regarding this field. Gossip running status is gossip thread running in C*. Where as STATUS field has different meaning. So we need to be clear that this is gossip thread running status